app/lib/lnd/lightning.js

```diff
@@ -173,11 +167,48 @@ class Lightning {
   // Helpers
   // ------------------------------------
 
+  /**
+   * Establish a connection to the Lightning interface.
+   */
+  async establishConnection(version: ?string) {
+    const { host, cert, macaroon } = this.lndConfig
+
+    // Find the rpc.proto file.. If no version was supplied, attempt to use the latest version.
+    this.version = version || (await getLatestProtoVersion())
```

Comment:
can't find where else `this.version` is used other than the next line. probably somewhere outside this module. In that case it may be useful to convert it into a getter in order to  have more control over it's value. If it's not used anywhere outside and is not meant to be used as such, consider converting it into a local const

Reply:
Correct - this is used outside of this module. I will make it a readonly getter.

Reply:
Updated.